src/main/java/com/backbase/ct/bbfuel/configurator/PaymentsConfigurator.java

```diff
@@ -57,8 +60,10 @@ public void ingestPaymentOrders(String externalUserId) {
 
             if (PAYMENT_TYPE_SEPA_CREDIT_TRANSFER.equals(paymentType)) {
                 randomArrangement = getRandomFromList(sepaCtArrangements);
-            } else {
+            } else if (PAYMENT_TYPE_US_DOMESTIC_WIRE.equals(paymentType)) {
                 randomArrangement = getRandomFromList(usDomesticWireArrangements);
+            } else {
```

Comment:
it's better to make this an `else if (PAYMENT_TYPE_ACH_DEBIT.equals(paymentType)) {`
and `throw new IllegalArgumentException("Unknown payment type " + paymentType);` in the `else` branch

Reply:
done :)